accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -437,17 +391,17 @@ int Function(int) func = (int x) => Foo(b).baz<int>(x);
 
 *Torn off extension methods are never equal unless they are identical*. Unlike instance methods, which are equal if it's the same method torn off from the same object (unless it's an instantiated tear-off of a generic function), torn off extension methods may close over the type variables of the extension as well. To avoid distinction between generic and non-generic extensions, no two torn off extension methods are equal, even if they are torn off from the same extension on the same object at the same static type.
 
-Extension methods torn off *constant* receiver expressions are not constant expressions. They also create a new function object each time the tear-off expression is evaluated.
+Extension methods torn off a *constant* receiver expressions are not constant expressions. They also create a new function object each time the tear-off expression is evaluated.
```

Comment:
Typo: `expression[s]`. Might read better as `An extension method torn off ...` in order to stay in singular everywhere.

Reply:
Made singularly correct.